daisy_workflows/build-publish/partner/cos_101_lts.publish.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Name": "cos-101-lts",
+  "WorkProject": "oslogin-staging-project",
+  "PublishProject": "oslogin-staging-project",
+  "ComputeEndpoint": "https://www.googleapis.com/compute/staging_alpha/projects/",
+  "DeleteAfter": "24h*30*6",
+  "Images": [
+    {
+      "Prefix": "cos-101-lts",
+      "Family": "cos-101-lts",
+      "Description": "COS 101 LTS",
+      "Architecture": "X86_64",
+      "GuestOsFeatures": ["VIRTIO_SCSI_MULTIQUEUE", "UEFI_COMPATIBLE", "SEV_CAPABLE"]
+    }
```

Comment:
I believe this needs GVNIC


gcloud compute images describe cos-101-17162-40-5 --project cos-cloud
architecture: X86_64
archiveSizeBytes: '1286280384'
creationTimestamp: '2022-09-19T15:54:40.509-07:00'
description: 'Google, Container-Optimized OS, 101-17162.40.5 LTS, Kernel: COS-5.15.65
  Kubernetes: 1.23.10 Docker: 20.10.12 Family: cos-101-lts, supports Shielded VM features'
diskSizeGb: '10'
family: cos-101-lts
guestOsFeatures:
- type: VIRTIO_SCSI_MULTIQUEUE
- type: UEFI_COMPATIBLE
- type: SEV_CAPABLE
- type: GVNIC
id: '7261753286452399583'

Reply:
Technically you're right. Although for the immediate purposes it doesn't matter. Will chat offline.